Sources/LiveViewNative/ViewTree.swift

```diff
@@ -93,14 +93,17 @@ struct ViewTreeBuilder<R: RootRegistry> {
         if let tagName = R.TagName(rawValue: element.tag) {
             R.lookup(tagName, element: element)
         } else {
-            BuiltinRegistry.lookup(element.tag, element, context: context)
-        }
+            BuiltinRegistry<R>.lookup(element.tag, element, context: context)
+        } 
     }
     
-    private func applyModifiers(encoded: String?, to view: some View, context: LiveContextStorage<R>) -> some View {
+    private func applyModifiers(encoded: String?, to view: some View, element: ElementNode, context: LiveContextStorage<R>) -> some View {
         let modifiers: [ModifierContainer<R>]
         if let encoded {
             let decoder = JSONDecoder()
+            decoder.userInfo[.liveContext!] = context
+            decoder.userInfo[.elementNode!] = element
+
```

Comment:
The context and element shouldn't be stored and passed through the decoder, they can be accessed from the modifier using their respective property wrappers:
```swift
struct MyModifier<R: RootRegistry>: ViewModifier, Decodable {
    @ObservedElement private var element
    @LiveContext<R> private var context
    // etc.
}
```